lua/astrocommunity/pack/typescript/init.lua

```diff
@@ -66,8 +66,10 @@ return {
           or util.root_has_file ".prettierrc.json5"
           or util.root_has_file ".prettierrc.js"
           or util.root_has_file ".prettierrc.cjs"
+          or util.root_has_file ".prettierrc.mjs"
           or util.root_has_file "prettier.config.js"
           or util.root_has_file "prettier.config.cjs"
+          or util.root_has_file "prettier.config.mjs"
```

Comment:
Could you follow the order of precedence here: 
https://prettier.io/docs/en/configuration.html
The .mjs files should be between the .js files and .cjs files